### ISSUE TITLE
Search-Elasticsearch-Cxn-NetCurl-6.80: test suite fails (with older Search::Elasticsearch?)

### ISSUE THREAD

**Issue by eserte** [elastic/elasticsearch-perl#183]
On some of my smoker systems:
```
#   Failed test 'No body - format'
#   at t/30_Logger/40_trace_request.t line 28.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1'
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true'
# '

#   Failed test 'Body - format'
#   at t/30_Logger/40_trace_request.t line 48.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# {
#    "foo" : "bar\n\u0027baz"
# }
# '
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# {
#    "foo" : "bar\n\u0027baz"
# }
# '
# '

#   Failed test 'Bulk - format'
#   at t/30_Logger/40_trace_request.t line 72.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# {"foo":"bar\n\u0027baz"}
# {"foo":"bar\n\u0027baz"}
# '
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# {"foo":"bar\n\u0027baz"}
# {"foo":"bar\n\u0027baz"}
# '
# '

#   Failed test 'Body string - format'
#   at t/30_Logger/40_trace_request.t line 94.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# The quick brown fox
# jumped over the lazy dog\u0027s basket'
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# The quick brown fox
# jumped over the lazy dog\u0027s basket'
# '
# Looks like you failed 4 tests of 11.
t/30_Logger/40_trace_request.t ................ 
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/11 subtests 
...
Undefined subroutine &Search::Elasticsearch::Role::Cxn::get_user_agent called at t/60_Cxn/30_http.t line 14.
t/60_Cxn/30_http.t ............................ 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```
This seems to happen if older Search::Elasticsearch (e.g. 6.00) is installed. Statistical analysis:
```
****************************************************************
Regression 'mod:Search::Elasticsearch'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	  18.95
[1='eq_6.80']  	      1.0000	      0.0000	38606834241837192.00

R^2= 1.000, N= 89, K= 2
****************************************************************
```

**Comment by ezimuel** [comment]
@eserte Thanks for reporting! I'm investigating more. I'll update you asap.

**Comment by ezimuel** [comment]
Fixed with #189
